src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/Columns/ColumnVisibility.jsx

```diff
@@ -2,6 +2,22 @@ import React from "react";
 import PropTypes from "prop-types";
```

Comment:
File comment

Reply:
Updated in 268ac92af166ed812fcb42ea5c7fc44b1466b4e2